jdbc/src/main/java/org/apache/zeppelin/jdbc/SqlCompleter.java

```diff
@@ -0,0 +1,250 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.zeppelin.jdbc;
+
+/*
+ * This source file is based on code taken from SQLLine 1.0.2 See SQLLine notice in LICENSE
```

Comment:
@jongyoul I think we need to update LICENSE here.

Quick search suggests that this origins of this code are from [Apache Hive](https://github.com/apache/hive/blob/master/LICENSE#L369) where they have

```
For the SQLLine package:

Copyright (c) 2002, 2003, 2004, 2005 Marc Prud'hommeaux

From: http://sqlline.sourceforge.net/#license

"SQLLine is distributed under the BSD License, meaning that you are free to redistribute, modify, or sell the software with almost no restrictions."

Statement from Marc Prud'hommeaux regarding inconsistent licenses in some SQLLine source files:

> SQLLine was once GPL, but it was changed to be BSD a few years back.
> Any references to the GPL are vestigial. Hopefully the license
> declaration at http://sqlline.sourceforge.net/#license is sufficiently
> authoritative in this regar
```

I would suggest that we add the same disclaimer to our [LICENSE](https://github.com/apache/incubator-zeppelin/blob/master/LICENSE#L251)

AFAIK [SQLLine](https://github.com/julianhyde/sqlline) is BSD-3-clauser now, so we also better add a [full text of the license](https://github.com/julianhyde/sqlline/blob/master/LICENSE) under `/licenses/`, as this code will be part of Apache source release.